cmd/kwil-cli/config/flags.go

```diff
@@ -45,9 +49,11 @@ func BindGlobalFlags(fs *pflag.FlagSet) {
 	fs.String(globalPrivateKeyFlag, cliCfg.PrivateKey, "the private key of the wallet that will be used for signing")
 	fs.String(globalProviderFlag, cliCfg.Provider, "the Kwil rpc provider HTTP endpoint")
 	fs.String(globalChainIDFlag, cliCfg.ChainID, "the expected/intended Kwil Chain ID")
+	fs.StringVar(&configFile, globalConfigFileFlag, defaultConfigFile, "the path to the Kwil CLI persistent global settings file")
 
 	// Bind flags to viper, named by the flag name
 	viper.BindPFlag(viperPrivateKeyName, fs.Lookup(globalPrivateKeyFlag))
 	viper.BindPFlag(viperProviderName, fs.Lookup(globalProviderFlag))
 	viper.BindPFlag(viperChainID, fs.Lookup(globalChainIDFlag))
+	viper.BindPFlag(viperConfigFile, fs.Lookup(globalConfigFileFlag))
```

Comment:
Do we still need to bind a viper setting to the flag?  I figured defining `viperConfigFile` wouldn't be necessary as I think those are for settings that appear in the config file?  Haven't tested; perhaps I misunderstand.

Reply:
let me verify this